vsdkx/addon/tracking/groupprocessor.py

```diff
@@ -0,0 +1,15 @@
+from vsdkx.core.structs import AddonObject
+
+from vsdkx.addon.tracking.processor import TrackerProcessor
+
+
+class GroupTrackerProcessor(TrackerProcessor):
+    """
+    Track group boxes
+    """
+    def post_process(self, addon_object: AddonObject) -> AddonObject:
+        addon_object.inference.extra["tracked_groups"], \
+        addon_object.shared["trackable_groups"] = \
```

Comment:
@nicolexirakia what should we call these two keys?

Reply:
@gujjs I really don't remember what we agreed on, I thought you kept notes on it. Probably one is 'detected_groups' and the other one 'tracked_groups'? But not 100% sure. Check the inference method one more time and see what happens to each one of these guys.

Reply:
also check the title of the task here: https://gitlab.com/natix/cvison/base/-/issues/111

Reply:
I don't think we agreed on anything. Nothing happens to these guys, according to you, they were logged before and never used

Reply:
that is whole another task that aims to rename existing used in production keys for the sake of better naming